indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -17,11 +17,8 @@ def get_add_handler(self):
         """Get the database handler
 
         """
-        try:
-            import redis
-            return redis.Redis()
-        except:
-            self.logger.error('Error importing Redis')
+        import redis
+        return redis.Redis()
```

Comment:
would it make sense to expose more parameters of Redis constructor (like connecting to a remote instance and so?)

Reply:
Yeah makes sense, I'll add them